### PR TITLE
 处理bit(1)无法转换成boolean-resolve bit(1) can't be boolean

### PR DESCRIPTION
**Pull request by qxl1231**
默认的5.3还是有问题,所以我自己处理

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-connector-mysql) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](https://loopback.io/doc/en/contrib/style-guide-es6.html)
- [ ] Commit messages are following our [guidelines](https://loopback.io/doc/en/contrib/git-commit-messages.html)

### CODE REVIEW

**Comment by slnode** [comment]
Can one of the admins verify this patch? To accept patch and trigger a build add comment ".*ok\W+to\W+test.*"

**Comment by qxl1231** [comment]
please be quick,many people are waiting for that, you can see the bug in stackover and issue, no version the mysql bit(1) still convert to Buffer!!!!!

**Comment by qxl1231** [comment]
@sam-github @rmg @sra please help us soon

**Comment by dhmlau** [comment]
@qxl1231, thanks for the PR.  There are a few things that we'd like you to take care first. Could you please:
- sign the CLA: https://cla.strongloop.com/agreements/strongloop/loopback-connector-mysql
- modify the commit message to be in English
- add tests to verify your changes.

Thanks!

**Comment by dhmlau** [comment]
@qxl1231, are you still interested in working on this PR? If so, could you please address my [above comment](https://github.com/strongloop/loopback-connector-mysql/pull/406#issuecomment-569381201)?  Thanks.

**Comment by jannyHou** [comment]
hi @qxl1231 are you still active working on this PR? I can help you add test cases and land it :) Thanks.

File: lib/mysql.js [hunk 1]
```diff
@@ -435,7 +435,10 @@ MySQL.prototype.fromColumnValue = function(prop, val) {
         }
         break;
       case 'Boolean':
-        val = Boolean(val);
+        val = Boolean(val);	        // BIT(1) case: <Buffer 01> for true and <Buffer 00> for false
```
Comment (jannyHou) [inline lib/mysql.js]: According to "默认的5.3还是有问题" is it only for a particular version like 5.3? If so we should check the db's version first, then apply the code.

**Comment by jannyHou** [comment]
I am closing this issue due to inactivity, sorry for the inconvenience and feel free to reopen it if you want to continue. 
Thanks for understanding.

**Pull request closed without merging**